### PR TITLE
delete individual blob files for completed file downloads to save space

### PR DESCRIPTION
**Pull request by akinwale**
Solution to save space until the "no blobs" feature is implemented on the daemon.

### CODE REVIEW

File: app/src/component/AppNavigator.js [hunk 1]
```diff
@@ -27,6 +27,7 @@ import {
   TextInput,
   ToastAndroid
 } from 'react-native';
+import { doDeleteCompleteBlobs } from '../redux/actions/file';
```
Comment (skhameneh) [inline app/src/component/AppNavigator.js]: Replace the relative path with an alias or a build resolve

File: app/src/constants.js [hunk 2]
```diff
@@ -1,10 +1,11 @@
...
-    KEY_FIRST_RUN_EMAIL: "firstRunEmail",
-    KEY_SHOULD_VERIFY_EMAIL: "shouldVerifyEmail",
+  KEY_FIRST_RUN_EMAIL: "firstRunEmail",
+  KEY_SHOULD_VERIFY_EMAIL: "shouldVerifyEmail",
 
-    SETTING_ALPHA_UNDERSTANDS_RISKS: "alphaUnderstandRisks",
+  SETTING_ALPHA_UNDERSTANDS_RISKS: "alphaUnderstandRisks",
 
-    ACTION_FIRST_RUN_PAGE_CHANGED: "FIRST_RUN_PAGE_CHANGED",
+  ACTION_DELETE_COMPLETED_BLOBS: "DELETE_COMPLETED_BLOBS",
```
Comment (skhameneh) [inline app/src/constants.js]: Why are some still mixed formatted? (camel vs underscores)

File: app/src/page/splash/index.js [hunk 3]
```diff
@@ -1,6 +1,7 @@
 import { connect } from 'react-redux';
 import { doBalanceSubscribe, doNotify } from 'lbry-redux';
 import { doAuthenticate, doUserEmailVerify, doUserEmailVerifyFailure, selectUser } from 'lbryinc';
+import { doDeleteCompleteBlobs } from '../../redux/actions/file';
```
Comment (skhameneh) [inline app/src/page/splash/index.js]: Replace the relative path with an alias or a build resolve

File: app/src/redux/actions/file.js [hunk 4]
```diff
@@ -10,6 +10,7 @@ import {
     selectDownloadingByOutpoint,
 } from 'lbry-redux';
 import { Alert, NativeModules } from 'react-native';
+import Constants from '../../constants';
```
Comment (skhameneh) [inline app/src/redux/actions/file.js]: Replace the relative path with an alias or a build resolve

File: app/src/constants.js (hunk 2 above)
Comment (akinwale) [reply app/src/constants.js]: Following convention with the constants defined in `lbry-desktop` and `lbry-redux`. The settings and keys are mixed case. The redux actions are uppercase with underscores.
https://github.com/lbryio/lbry-redux/blob/master/src/constants/action_types.js
https://github.com/lbryio/lbry-redux/blob/master/src/constants/settings.js

**Pull request merged**